extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -164,6 +174,23 @@ class AmpLightbox extends AMP.BaseElement {
   }
 
   /**
+   * reschedule layout on animation/transition end. But ignore the first transitionend event
+   * @param {!Event} event
+   */
+  reschedule_(event) {
+    if (event.type == 'transitionend') {
+      // Ignore the first transitionend event.
```

Comment:
a) I found it hard to use event property here. 'target' is not a good property to check since transition apply to lightbox element/ iframe element/iframe child element can all result in iframe size change.
b) c). Yes! I agree an extra `scheduleLayout` doesn't hurt. changed.